src/ocean_component.cpp

```diff
@@ -208,26 +209,19 @@ void OceanComponent::prepareToRun() {
   const double spy = 60 * 60 * 24 * 365.25; // seconds per year
 
   // ocean depth
-  double thick_LL =
+  const double thick_LL =
```

Comment:
Why is the const double call necessary here? I get that the think_LL value should not change does the const double declaration protect the value from being changed else where in the code?